tests/client_dtypes_test.py

```diff
@@ -119,3 +121,41 @@ def test_ipv4_normalization(self):
         ipv4 = ak.IPv4(ip_list)
         ip_as_int = ipv4.normalize("192.168.1.1")
         self.assertEqual(3232235777, ip_as_int)
+
+    def test_is_ipv4(self):
+        x = [random.getrandbits(32) for i in range(100)]
+
+        ans = ak.is_ipv4(ak.array(x, dtype=ak.uint64))
+        self.assertListEqual(ans.to_ndarray().tolist(), [True]*100)
```

Comment:
@pierce314159 and I have been trying to use `assertListEqual` more because in the event that something fails, it gives information on the indexes as opposed to just stating failed. Not as clean in appearance, but has been helpful in debugging.